test/cql-pytest/test_using_timestamp.py

```diff
@@ -90,3 +90,79 @@ def test_key_writetime(cql, table1):
         cql.execute(f'SELECT writetime(k) FROM {table1}')
     with pytest.raises(InvalidRequest, match='PRIMARY KEY part k'):
         cql.execute(f'SELECT ttl(k) FROM {table1}')
+
+# Reproducer for https://github.com/scylladb/scylladb/issues/14182
```

Comment:
If you already bothered to write a paragraph-long explanation of this test (and the second test) in the commit message, it will be great to copy it here. So in a few years if this test fails, the reader won't need to start guessing what this test was supposed to check.

Nitpick: It's great that you mentioned the issue that this reproduces, but you can also write "#14182" instead of the long github link.

Reply:
I dislike the `#14182` as it implies the scylladb.git repo.
This breaks when this patch is merged into the scylla-enterprise repo.